app/routes/coc._index.tsx

```diff
@@ -9,104 +9,136 @@ export const meta: V2_MetaFunction = () => {
 export default function Index() {
   return (
     <Layout>
-      <article className="prose prose-slate mx-auto max-w-3xl py-10 dark:prose-invert lg:prose-xl">
-        <h2>Code of Conduct</h2>
+      {/* <article className="prose prose-slate mx-auto w-[300px] py-10 dark:prose-invert lg:prose-xl prose-h3:my-0 sm:w-[600px] sm:max-w-xl md:max-w-3xl lg:w-[1000px]"> */}
```

Comment:
wondering why is there a commented line here? if its no longer used, let's delete it.

Reply:
design responsiveness all perfect, once its updated, please merge @jeeehaan

Reply:
> wondering why is there a commented line here? if its no longer used, let's delete it.

Sorry, I forgot to delete it, will do it soon.

Reply:
> design responsiveness all perfect, once its updated, please merge @jeeehaan

Will do, thank you for the review mas.